lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -247,22 +343,11 @@ module.exports = function({workspace, dependencies, options: {projectName}}) {
 					const libraryNamespaceMatch = libraryIndicatorPath.match(libraryNamespacePattern);
 					if (libraryNamespaceMatch && libraryNamespaceMatch[1]) {
 						const libraryNamespace = libraryNamespaceMatch[1];
-						return moduleBundler({
-							options: {
-								bundleDefinition: getBundleDefinition(libraryNamespace),
-								bundleOptions: {
-									optimize: true,
-									usePredefineCalls: true,
-									ignoreMissingModules: true
-								}
-							},
-							resources
-						}).then(([bundle]) => {
-							if (bundle) {
-								// console.log(`${libraryNamespace}/library-preload.js bundle created`);
-								return workspace.write(bundle);
-							}
-						});
+						return createLibraryBundles(libraryNamespace, resources)
+							.then((results) => {
+								const bundles = Array.prototype.concat.apply([], results);
+								return Promise.all(bundles.map((bundle) => bundle && workspace.write(bundle)));
```

Comment:
Due to the new `skipIfEmpty`, some of the entries in the `bundles` array might be `undefined`.